docker/compose/mozdef_meteor/Dockerfile

```diff
@@ -29,23 +29,23 @@ RUN \
 RUN \
     curl -sL https://rpm.nodesource.com/setup_8.x | bash - && \
     yum -y install nodejs
-RUN \
-  mkdir /opt/mozdef/meteor && \
-  curl -sL -o /opt/mozdef/meteor.tar.gz https://static-meteor.netdna-ssl.com/packages-bootstrap/$METEOR_VERSION/meteor-bootstrap-os.linux.x86_64.tar.gz && \
-  tar -xzf /opt/mozdef/meteor.tar.gz -C /opt/mozdef/meteor && \
-  mv /opt/mozdef/meteor/.meteor /opt/mozdef && \
-  rm -r /opt/mozdef/meteor && \
-  cp /opt/mozdef/.meteor/packages/meteor-tool/*/mt-os.linux.x86_64/scripts/admin/launch-meteor /usr/bin/meteor
+
+RUN mkdir /opt/mozdef/meteor
```

Comment:
I thought we want to make these lines all under one RUN command, so that just one RUN command caches all these results (the filesize of the container is smaller as a result).

Reply:
#wontfix